farm-22-04/scripts/012-farm.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# Create the farm user
+useradd --home-dir /home/farm \
+        --shell /bin/bash \
+        --create-home \
+        --system \
+        farm
+
+# Setup the home directory
+chown -R farm: /home/farm
+
+# Setup
+chown -R farm: /var/www/html
+
+usermod -aG sudo farm
+
+# Create farm folder
+mkdir /home/farm
```

Comment:
No need to explicitly create `/home/farm`
It is already created during `useradd`